tests/test_fft_forward_real/LICENSE.txt

```diff
@@ -1,6 +1,6 @@
 Software Release License Agreement
 
-Copyright (c) 2016-2017, XMOS, All rights reserved.
+Copyright (c) 2016-2020, XMOS, All rights reserved.
```

Comment:
2020?

Reply:
The original commit changing the folder with the license was done in 2020, this is the reason why 2020 was there and the license check didn't fail.

I have updated the date to 2021.